src/sync/outgoingSyncUtils.js

```diff
@@ -192,6 +191,8 @@ function generateSyncData(settings, recordType, record) {
         category_ID: record.category && record.category.id,
         confirm_time: getTimeString(record.confirmDate),
         store_ID: settings.get(THIS_STORE_ID),
+        requisition_ID: record.isLinkedToRequisition ?
```

Comment:
@edmofro if `record.linkedRequisition` is _false_, then it'll assign _false_ to `requisition_ID` rather than null/undefinted